FirebaseRemoteConfig/Tests/SwiftUnit/UserDefaultsManagerOrigTests.swift

```diff
@@ -0,0 +1,242 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@testable import FirebaseRemoteConfig
+import XCTest
+
+private let appName = "testApp"
+private let fqNamespace1 = "testNamespace1:testApp"
+private let fqNamespace2 = "testNamespace2:testApp"
+private var userDefaultsSampleTimestamp: TimeInterval = 0
+
+class UserDefaultsManagerOrigTests: XCTestCase {
+  override func setUp() {
+    super.setUp()
+    // Clear UserDefaults before each test.
+    UserDefaults.standard.removePersistentDomain(forName: Bundle.main.bundleIdentifier!)
+    userDefaultsSampleTimestamp = Date().timeIntervalSince1970
+  }
+
+  func testUserDefaultsEtagWriteAndRead() {
+    let manager = UserDefaultsManager(appName: appName,
+                                      bundleID: Bundle.main.bundleIdentifier!,
```

Comment:
Optional: make bundleID a property that gets initialized in setup to avoid the force unwrap in every test.

Reply:
Done.